src/main/java/core/basesyntax/io/impl/CsvTransactionRequestDeserializer.java

```
@@ -0,0 +1,42 @@
+package core.basesyntax.io.impl;
+
+import core.basesyntax.io.TransactionRequestDeserializer;
+import core.basesyntax.model.FruitType;
+import core.basesyntax.model.FruitTypeFactory;
+import core.basesyntax.model.TransactionRequest;
+import core.basesyntax.model.TransactionType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvTransactionRequestDeserializer implements TransactionRequestDeserializer {
+    private final FruitTypeFactory fruitTypeFactory;
+
+    public CsvTransactionRequestDeserializer(FruitTypeFactory fruitTypeFactory) {
+        this.fruitTypeFactory = fruitTypeFactory;
+    }
+
+    @Override
+    public List<TransactionRequest> loadFrom(Path path) {
+        try (BufferedReader reader = Files.newBufferedReader(path)) {
+            return reader.lines().skip(1).map(this::parseLine).collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

The catch block rethrows `new RuntimeException(e)` without context. This violates the checklist item: "Don't ignore exceptions in try-catch blocks" and the guidance: "Exception messages should be informative (include context like file path, line content, or operation/fruit causing the error)". Rethrow with a message that includes the `path` (e.g., "Failed to read transactions from <path>: <cause>") and pass the original exception as the cause.